Modules/Loadable/SubjectHierarchy/Widgets/qMRMLSubjectHierarchyComboBox.h

```diff
@@ -56,6 +56,17 @@ class Q_SLICER_MODULE_SUBJECTHIERARCHY_WIDGETS_EXPORT qMRMLSubjectHierarchyCombo
   /// If aligned, the popup will shift vertically so that the selected item overlays above the combobox.
   /// Else, the popup tree appears below the combobox, like for a qMRMLNodeComboBox.
   Q_PROPERTY(bool alignPopupVertically READ alignPopupVertically WRITE setAlignPopupVertically)
+  /// This property controls whether an extra item is added before any subject hierarchy item under
+  /// the scene item for indicating 'None' selection.
+  Q_PROPERTY(bool noneItemEnabled READ noneItemEnabled WRITE setNoneItemEnabled)
+  /// This property controls the name that is displayed for the None item.
+  /// "None" by default.
+  /// \sa noneItemEnabled
+  Q_PROPERTY(QString noneItemText READ noneItemText WRITE setNoneItemText)
+  /// This property controls whether hierarchy information is included in the title text.
+  /// If enabled (which is the default), then the title typically looks like "[ParentName] / [ParentName] / [SelectedItemName]".
+  /// If disabled, the title will only be "[SelectedItemName]".
+  Q_PROPERTY(bool showParentsInTitle READ showParentsInTitle WRITE setShowParentsInTitle)
```

Comment:
It is not really a title. In a Qt combo-box it is called the "lineEdit", which might not the most intuitive name either, but at least not we invented the name just use an existing name.

Reply:
I just tried to follow the convention I found. See `updateComboBoxTitleAndIcon` function. At least it is definitely not a non-existent name.

Reply:
`updateComboBoxTitleAndIcon` is not in the public API, so it was not important there to have a meaningful or consistent name. I don't have much better suggestions, so if you don't like any of the ones that I listed below any better then I'm OK with the current name, too.

showCurrentItemParents
showCurrentItemParentsInLineEdit
showParentsInLineEdit

Reply:
I renamed this to `showCurrentItemParents`. I think LineEdit is even worse than Title because it cannot be edited in this case.